CP-CPS.md

```diff
@@ -228,9 +228,7 @@ No stipulation.
 
 ### 3.2.2 Authentication of organization identity
 
-ISRG only issues Domain Validation (DV) certificates. All FQDNs which will be listed in the Common Name and list of SANs in the certificate are fully validated prior to issuance.
-
-ISRG uses three methods for validating domain control:
+Prior to issuance of a Subscriber Certificate, ISRG uses at least one of the following methods to validate the Applicant's control of each requested FQDN:
```

Comment:
Should we allow ourselves to issue IP Address certs here? Or maybe in another change

Reply:
Yeah, I'd like that to be a follow-up change. We [have a bug tracking that](https://github.com/letsencrypt/cp-cps/issues/207), and I wouldn't mind publishing it in this same CPS update, but I'd at least like to keep it separate from this PR.